apps/studio/components/interfaces/Database/Functions/CreateFunction/index.tsx

```diff
@@ -68,15 +69,15 @@ const FormSchema = z.object({
     .optional(),
 })
 
-const CreateFunction = ({ func, visible, setVisible }: CreateFunctionProps) => {
+const CreateFunction = ({ func, visible, isDuplicating = false, onClose }: CreateFunctionProps) => {
   const { data: project } = useSelectedProjectQuery()
   const [isClosingPanel, setIsClosingPanel] = useState(false)
   const [advancedSettingsShown, setAdvancedSettingsShown] = useState(false)
   // For now, there's no AI assistant for functions
   const [assistantVisible, setAssistantVisible] = useState(false)
```

Comment:
Can I also delete this? Its not being used at all and comments no AI assistant is visible for function

Reply:
Yeah, let's just delete it